### DISCUSSION TITLE
Mono: how to have icons look like the demo in menus?

### DISCUSSION THREAD

**Discussion by warpdesign** [GetPublii/Publii#1360] [Help] [answered]
The [preview](https://demo.getpublii.eu/themes/mono/) page of the Mono theme shows nice SVG icons that have no fill color (generated svg has `fill="none"`): how can I make my site look like this? By default the fill property is set, and I see no way to change it in the shortcode, nor in the feather icons plugin settings.

### Mono demo page:
![image](https://github.com/GetPublii/Publii/assets/199648/889ddebd-5923-4796-903b-928b4542e1e0)

SVG generated: (notice the `fill="none"`)
`<svg class="fi fi-home" width="24px" height="24px" fill="none" stroke="currentColor" stroke-width="2">`

### My website:
![image](https://github.com/GetPublii/Publii/assets/199648/55c117a1-3362-4848-97f4-8cf1dc5ba7f0)

SVG generated: (notice there's no `fill="none"`)
`<svg class="fi fi-home" width="24px" height="24px" stroke="currentColor" stroke-width="2" viewBox="0 0 24 24">`

**Comment by bobmitro** [comment]
Are you using the latest version of the Feather Icon plugin (v. 1.1.0)? The fill="none" attribute is added by default to the icons.

**Reply by warpdesign** [reply]
I have installed version 1.1.0: any idea why it wouldn't work as expected?

**Comment by bobmitro** [comment, marked as answer]
My apologies, I forgot to upload the new version of the plugin with the fill="none" attribute to the download section. Please download version 1.2.0, and it should work now.

**Reply by warpdesign** [reply]
What's the prefered way to update a plugin?
- Install new version over the previous one?
- uninstall then install new version?

**Reply by internettips** [reply]
I just drag the new plugin .zip file and place it over the top of the existing plugin. So similar process to installing a Publii theme.
